examples/bow.py

```diff
@@ -55,7 +55,7 @@ def batcher(params, batch):
             if word in params.word_vec:
                 sentvec.append(params.word_vec[word])
         if not sentvec:
-            vec = np.zeros(len(params.word_vec.values()[0]))
+            vec = np.zeros(len(list(params.word_vec.values())[0]))
```

Comment:
This will be super slow, it will transform the params.word_vec.values() to a list at every call of the batcher. Let's compute the dimension of the word vectors (which is actually always 300...) in the "prepare" function only once instead.